docs/code_coverage.md

```diff
@@ -87,20 +87,27 @@ python infra/helper.py profile --fuzz-target=<fuzz_target_name> --corpus-dir=<my
 ### Additional arguments for `llvm-cov`
 
 You may want to use some of the options of [llvm-cov tool], for example,
-`-ignore-filename-regex=` or `-tab-size=`. You can pass those to the helper
-script after `--`:
+`-ignore-filename-regex=`. You can pass those to the helper script after `--`:
 
 ```bash
-python infra/helper.py profile $project_name -- -ignore-filename-regex='.*code/to/be/ignored/.*' -tab-size=2
+python infra/helper.py profile $project_name -- -ignore-filename-regex=.*code/to/be/ignored/.* <other_extra_args>
 ```
 
-To specify particular source files to be shown in the report, list the filepaths
-at the end of the extra arguments sequence, for example:
+To specify particular source files or directories to be shown in the report,
+list the paths at the end of the extra arguments sequence, for example:
 
 ```bash
-python infra/helper.py profile zlib -- -tab-size=8 /src/zlib/inftrees.c /src/zlib_uncompress_fuzzer.cc /src/zlib/zutil.c
+python infra/helper.py profile zlib -- <other_extra_args> /src/zlib/inftrees.c /src/zlib_uncompress_fuzzer.cc /src/zlib/zutil.c
 ```
 
+If you want OSS-Fuzz to use some extra arguments when generating code coverage
```

Comment:
It's somewhat awkward that <other_extra_args> isn't explained before it is used, but I don't have any suggested improvements.